includes/payments/backward-compatibility.php

```diff
@@ -0,0 +1,632 @@
+<?php
```

Comment:
@DevinWalker I added backward compatibility for:
1. Old meta keys
2. New meta key before update
3. SQL query support for old meta keys